spec/lib/audit/checksum_spec.rb

```diff
@@ -137,4 +136,52 @@
       described_class.validate_list_of_druids(csv_file_path)
     end
   end
+
+  describe '.validate_status_root' do
+    include_context 'fixture moabs in db'
+    let(:subject) { described_class.validate_status_root('validity_unknown', endpoint_name, limit) }
+
+    context 'when there are PreservedCopies to check' do
+      let(:cv_mock) { instance_double(ChecksumValidator) }
+
+      it 'creates an instance and calls #validate_checksums for every result when results are in a single batch' do
+        allow(ChecksumValidator).to receive(:new).and_return(cv_mock)
+        expect(cv_mock).to receive(:validate_checksums).exactly(3).times
+        subject
```

Comment:
FYI, using `subject` for a time-dependent execution like this is something of an anti-pattern.  Since it is only invoked once, it is probably clearer to make the `.validate_status_root` call directly here.

Reply:
will do!

Reply:
and done ...